### PR TITLE
[DEM-208] We are missing the YES , No page

### PR DESCRIPTION
**Pull request by vietnogi**
ping @appirio-tech/demandside

### CODE REVIEW

File: src/client/app/submit-work/about-project/views/brief.jade [hunk 1]
```diff
@@ -1,8 +1,27 @@
...
+
+form(ng-show="vm.showBrief")
+  label(for="brief-your-project") Please upload your project brief.
 
-form
   input#brief-your-project(type="file", placeholder="My Project", ng-model="vm.briefFilename")
 
   next-state(state="vm.nextState", vm="vm")
+
+form(name="summaryForm", ng-show="vm.showElevator")
```
Comment (tladendo) [inline src/client/app/submit-work/about-project/views/brief.jade]: Why is this page duplicated? I understand the logic here, but if we're going to do it this way, we need to bring in the elevator pitch controller and delete the duplicated template.

Comment (tladendo) [inline src/client/app/submit-work/about-project/views/brief.jade]: sorry @vietnogi -- missed your chats. Want me to just do a PR against this and then we can merge in?

Comment (vietnogi) [inline src/client/app/submit-work/about-project/views/brief.jade]: yep, that was in the chat too ^.^

Comment (vietnogi) [inline src/client/app/submit-work/about-project/views/brief.jade]: also, created a lira ticket for you

Comment (tladendo) [inline src/client/app/submit-work/about-project/views/brief.jade]: grazie

**Comment by indytechcook** [comment]
looks good

**Pull request merged**